src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
Apple doesn't support generating DSA keys via SecKeyGeneratePair (they can do it via the CDSA library, but it's been marked as deprecated for many years).  If Apple were to add the support we could do so easily.  If they don't, we'd have to do a lot of work for an algorithm that no one really cares about anymore.  (The same spec that let DSA use the SHA-2 family of hashes produced ECDSA, and people seemed happier to jump to ECC than revamp their DSA implementations and usage... but this parenthetical is an observation, not a fact :smile:).